drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/ModelSynchronizerImpl.java

```diff
@@ -205,7 +205,8 @@ public void deleteCell( final GridData.Range rowRange,
     @SuppressWarnings("unchecked")
     public void appendColumn( final BaseColumn column ) throws MoveColumnVetoException {
         final MetaData metaData = new BaseColumnSynchronizer.ColumnMetaDataImpl( column );
-        for ( Synchronizer synchronizer : synchronizers ) {
+        for ( int i = 0; i < synchronizers.size(); i++ ) {
+            final Synchronizer synchronizer = synchronizers.get( i );
```

Comment:
@manstis Any obscure reason to make this change? Seems like a slower iterator on a List.